internal/service/verify/resource_verify_policy_test.go

```diff
@@ -0,0 +1,523 @@
+		resource.TestCheckResourceAttr(resourceFullName, "phone.create_mfa_device", "true"),
+		resource.TestCheckResourceAttr(resourceFullName, "phone.otp.attempts.count", "2"),
+		resource.TestCheckResourceAttr(resourceFullName, "phone.otp.lifetime.duration", "7"),
+		resource.TestCheckResourceAttr(resourceFullName, "phone.otp.lifetime.time_unit", "MINUTES"),
+		resource.TestCheckResourceAttr(resourceFullName, "phone.otp.deliveries.count", "3"),
+		resource.TestCheckResourceAttr(resourceFullName, "phone.otp.deliveries.cooldown.duration", "16"),
+		resource.TestCheckResourceAttr(resourceFullName, "phone.otp.deliveries.cooldown.time_unit", "SECONDS"),
+		resource.TestCheckResourceAttr(resourceFullName, "phone.otp.notification.template_name", "email_phone_verification"),
+		resource.TestCheckResourceAttr(resourceFullName, "phone.otp.notification.variant_name", "variant23_b"),
+
+		resource.TestCheckResourceAttr(resourceFullName, "transaction.timeout.duration", "27"),
+		resource.TestCheckResourceAttr(resourceFullName, "transaction.timeout.time_unit", "MINUTES"),
+		resource.TestCheckResourceAttr(resourceFullName, "transaction.data_collection.timeout.duration", "12"),
+		resource.TestCheckResourceAttr(resourceFullName, "transaction.data_collection.timeout.time_unit", "MINUTES"),
+		resource.TestCheckResourceAttr(resourceFullName, "transaction.data_collection_only", "false"),
+
+		resource.TestMatchResourceAttr(resourceFullName, "created_at", validation.RFC3339Regexp),
+		resource.TestMatchResourceAttr(resourceFullName, "updated_at", validation.RFC3339Regexp),
+	)
+
+	minimalPolicy := resource.ComposeTestCheckFunc(
+		resource.TestMatchResourceAttr(resourceFullName, "id", validation.P1ResourceIDRegexp),
+		resource.TestMatchResourceAttr(resourceFullName, "environment_id", validation.P1ResourceIDRegexp),
+		resource.TestCheckResourceAttr(resourceFullName, "name", updatedName),
+		resource.TestCheckResourceAttr(resourceFullName, "description", updatedName),
+		resource.TestCheckResourceAttr(resourceFullName, "default", "false"),
+
+		resource.TestCheckResourceAttr(resourceFullName, "government_id.verify", "REQUIRED"),
+
+		resource.TestCheckResourceAttr(resourceFullName, "facial_comparison.verify", "DISABLED"),
+		resource.TestCheckResourceAttr(resourceFullName, "facial_comparison.threshold", "MEDIUM"),
+
+		resource.TestCheckResourceAttr(resourceFullName, "liveness.verify", "DISABLED"),
+		resource.TestCheckResourceAttr(resourceFullName, "liveness.threshold", "MEDIUM"),
+
+		resource.TestCheckResourceAttr(resourceFullName, "email.verify", "DISABLED"),
+		resource.TestCheckResourceAttr(resourceFullName, "email.create_mfa_device", "false"),
+		resource.TestCheckResourceAttr(resourceFullName, "email.otp.attempts.count", "5"),
+		resource.TestCheckResourceAttr(resourceFullName, "email.otp.lifetime.duration", "10"),
+		resource.TestCheckResourceAttr(resourceFullName, "email.otp.lifetime.time_unit", "MINUTES"),
```

Comment:
Is it possible to vary the time units also?  Unless this is fixed at `MINUTES`

Reply:
Test cases expanded.  Resolved.